src/promptflow/tests/sdk_cli_test/e2etests/test_experiment.py

```diff
@@ -136,6 +139,9 @@ def _assert_result(result):
             assert expected_output_path.resolve().exists()
             # Assert eval metric exists
             assert (expected_output_path / "eval" / "flow.metrics.json").exists()
+            # Assert session exists
+            line_runs = client._traces.list_line_runs(session_id=session)
+            assert len(line_runs) > 0
```

Comment:
test session also in non-experiment test?

Reply:
Yes, and only for user who set the 'enable_internal_features'=True for now.